### ISSUE TITLE
Can't register the plugin: `Plugin failed to decode: invalid type: string "Any", expected a sequence`

### ISSUE THREAD

**Issue by cig0** [cptpiepmatz/nu-plugin-highlight#3]
Nushell: `0.83.0`
cargo: `1.70.0`
nu_plugin_highlight:  `v1.0.3`

```
> whereis nu_plugin_highlight
nu_plugin_highlight: /var/home/cig0/.cargo/bin/nu_plugin_highlight

> register ~/.cargo/bin/nu_plugin_highlight
Error:   × Error getting signatures
   ╭─[entry #3:1:1]
 1 │ register ~/.cargo/bin/nu_plugin_highlight
   · ────┬───
   ·     ╰── Plugin failed to decode: invalid type: string "Any", expected a sequence
   ╰────
```

If running the plugin 'as-is':
```
> nu_plugin_highlight
msgpack
��Error��Plugin failed to decode�>invalid value: integer `10`, expected variant index 0 <= i < 3�
```

I couldn't find anything related on the web.
Thanks.

**Comment by cptpiepmatz** [comment]
I personally did not update my nushell yet, I will have a look a this tomorrow

**Comment by cptpiepmatz** [comment]
Should be fixed in [v1.0.4](https://github.com/cptpiepmatz/nu-plugin-highlight/releases/tag/v1.0.4).
